### DISCUSSION TITLE
newbie questions

### DISCUSSION THREAD

**Discussion by eccovirtua** [Glyceri/FFXIVPetRenamer#9] [Q&A] [answered]
is it bannable? is it just for me or it can it be seen by other ppl using or not using the plugin

**Comment by Glyceri** [comment, marked as answer]
Only you can see pet names. You can share your names with others so you can see each others names. But that is only to people you manually share with. If you go live streaming with it, then yes, it's bannable. Like any other plugin.

**Comment by eliwilltechdiff** [comment]
When I open the pet rename window there doesn't seem to be anything to click on to rename my minion, and when I open the pet list none of my minions show up and I don't know how to add them. The "Give Nickname" button in the top right just takes me to those two windows, but I don't get the ability to actually rename the minion. Am I missing something? 
![image](https://github.com/user-attachments/assets/c5fce891-1b41-4b51-bac5-780b16b42ece)

**Reply by Glyceri** [reply]
Hello hellooo!
In the `Pet Rename Window` at the `bottom` there is an input box.
The layout is like this
`[Nickname:][Save Icon][Clear Icon][|                                ]` <-- this is an input box
If you type a name in the input box and press enter or the save button your pet will get the nickname.

The petlist is meant to see all the pets you have given a nickname.

**Reply by Glyceri** [reply]
<img width="560" alt="image" src="https://github.com/user-attachments/assets/ba772ec5-0deb-4145-9724-19e0867af65d">

Its probably easier if I show an image

**Reply by eliwilltechdiff** [reply]
> <img alt="image" width="560" src="https://private-user-images.githubusercontent.com/78602618/378173802-ba772ec5-0deb-4145-9724-19e0867af65d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk0NDE2ODcsIm5iZiI6MTcyOTQ0MTM4NywicGF0aCI6Ii83ODYwMjYxOC8zNzgxNzM4MDItYmE3NzJlYzUtMGRlYi00MTQ1LTk3MjQtMTllMDg2N2FmNjVkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDIwVDE2MjMwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ5Nzc5ODIyNDNiMjQxYzMxMGIwNTVlNDFjYWRhMGZlYjZlOTg1ZmE2MTI4OTlmNDc0MDIyNjFmZDQ1ZDc4NWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.lll7K8bz54GhUmqajgV2jeU6p8HwQkVPGlzDfbn9p0M">
> Its probably easier if I show an image

I may be stupid :p but thank you
